src/common/src/util/value_encoding/mod.rs

```diff
@@ -31,10 +31,7 @@ use error::ValueEncodingError;
 pub fn serialize_cell(cell: &Datum) -> Result<Vec<u8>> {
```

Comment:
Can we remove `*_not_null` after removing null tag, since the only difference is that `*_not_null` will provide some check.

Reply:
cc @skyzh

Reply:
Btw, the lib `src/utils/value-encoding` became useless. I think we can remove it or move `src/common/src/util/value_encoding` into it. But since we don't use Serde for value encoding, maybe just removing it is better?

Reply:
I think all of data in our system is nullable by now. We can remove `*_not_null` indeed, and only provide ways to serialize nullable data.

Reply:
I think it's worth having a crate solely for value encoding. Maybe we will expand it in the future?